ddl/ddl_worker.go

```diff
@@ -1361,7 +1369,7 @@ func updateSchemaVersion(d *ddlCtx, t *meta.Meta, job *model.Job, multiInfos ...
 			if droppedIDs, ok := job.CtxVars[0].([]int64); ok {
 				if addedIDs, ok := job.CtxVars[1].([]int64); ok {
 					// to use AffectedOpts we need both new and old to have the same length
-					maxParts := mathutil.Max[int](len(droppedIDs), len(addedIDs))
+					maxParts := mathutil.Max(len(droppedIDs), len(addedIDs))
```

Comment:
Seem this line code is no related with pause|resume problem? please do not introduce non-related code change.